### DISCUSSION TITLE
Estimation of transition delay distributions from consecutive surveillance time series

### DISCUSSION THREAD

**Discussion by pitmonticone** [epiforecasts/EpiNow2#226] [Q&A] [answered]
Hi, 

First of all I would like to thank you very much for the active development of this great package!

Now, suppose you have 

* a pair of consecutive age-stratified surveillance datasets `(primary_incidence, secondary_incidence)` where `primary_incidence` might be the time series of symptomatic cases by date of symptom onset, hospital admission by date of admission or ICU admission by date of admission so that `secondary_incidence` would be the series  of hospital admissions by date of admission, ICU admissions by date of admission or deceased cases by date of death respectively 
* a collection of estimated age-specific transition rates from the literature (e.g. symptomatic fraction, Infection hospitalization ratio, sero-adjusted infection fatality ratio).

Would it be sensible to try to estimate transition delay distributions from `(primary_incidence, secondary_incidence)`? If yes, then what method would you recommend ? Are you thinking about implementing something similar in the near future ?    

Thanks.

**Comment by ClaudMor** [comment]
Nice question @pitmonticone ! I would be very interested too... Does anybody have an answer about this?

**Comment by seabbs** [comment, marked as answer]
Hi all, 

Apologies for missing this. 

See my thoughts on this here (which I am happy to explore more): https://github.com/epiforecasts/EpiNow2/issues/229#issuecomment-780680176
